src/HLLC.hpp

```diff
@@ -36,7 +36,10 @@ AMREX_FORCE_INLINE AMREX_GPU_DEVICE auto HLLC(quokka::HydroState<N_scalars> cons
 	double cs_tilde = NAN;
 	if (gamma != 1.0) {
 		// TODO(ben): implement Roe average for general EOS
-		cs_tilde = std::sqrt((gamma - 1.) * (H_tilde - 0.5 * vsq_tilde));
+		const double C_star_P = 1.0 / (gamma - 1.0);
```

Comment:
@BenWibking - the `HydroState` struct that is used in HLLC does not store massScalars. It only stores the total scalars. Do we need to add massScalars to this struct? Because we need to access massScalars in HLLC and we can't use our `ComputeMassScalars` function from `RadSystem` here.

Reply:
Yeah, I think it makes sense to add massScalars to the HydroState.